lib/typeErrors.ml

```diff
@@ -276,6 +277,9 @@ let pp_welltyped = function
       ^^^ !^(string_of_int has)
     in
     { short; descr = Some descr; state = None }
+  | Unexpected_computational_args_in_lemma ->
+    let short = !^"Unexpected computational arguments in lemma definition" in
+    { short; descr = None; state = None }
```

Comment:
It looks like you don't use this anywhere?  Also this is not a user-facing error, it would be an internal development error, so I don't think it merits a message - you may delete it.

Reply:
Ok so earlier in the file there's a `failwith "unexpected computational argument in lemma"` so I'm confused